corehq/apps/export/templates/export/partials/export_list_controller.html

```diff
@@ -29,10 +29,15 @@
         {% endblocktrans %}
       </a>
     </p>
-    {% if not odata_feeds_over_limit %}
+    {% if not feeds_over_limit %}
       {% include 'export/partials/export_list_create_export_modal.html' %}
     {% else %}
-      {% include 'export/partials/odata_feed_limit_reached_modal.html' %}
+      {% if isOData %}
+        {% include 'export/partials/odata_feed_limit_reached_modal.html' %}
```

Comment:
i think the URLs to these partials might need to be updated?

Reply:
also need to ensure that they get cleaned up / removed if they are no longer being used

Reply:
@RynhardPietersen bump on this one